packages/components/src/components/vertical-tab/vertical-tab-layout.jsx

```diff
@@ -11,4 +11,4 @@ const VerticalTabLayout = ({ children, is_full_width }) => (
     </div>
 );
 
-export { VerticalTabLayout };
+export default VerticalTabLayout;
```

Comment:
Please prefix all classnames in this file with `dc-` (with updates to their respective stylesheet).